index.html

```diff
@@ -1,96 +1,144 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="utf-8">
 	<title>Home</title>
 	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
 	<link rel="stylesheet" href="css/index.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<!--[if IE]>
```

Comment:
Adding the vieport meta tag means you already thinking ahead in your web application responsiveness.